gp-includes/assets-loader.php

```diff
@@ -60,7 +60,7 @@ function gp_enqueue_styles( $handles ) {
  * @param string $handle A single style handle to enqueue.
  */
 function gp_enqueue_style( $handle ) {
-	if ( ! in_array( $handle, GP::$styles ) ) {
+	if ( false === in_array( $handle, GP::$styles ) ) {
```

Comment:
What's the reason for this change? Did you mean to change it to`! in_array( $handle, GP::$styles, true )`?